src/widgets/CustomAttributesViewer.php

```diff
@@ -33,6 +33,7 @@ public function run(): string
                 ],
                 'value' => [
                     'attribute' => 'value',
+                    'format' => 'html',
```

Comment:
Use explicit `Html::encode` on values that you output.
`format => html` is a sanitizer for untrusted user input: it consumes a lot of resources compared to encoding